lib/God/ForkMode.js

```diff
@@ -105,7 +105,6 @@ module.exports = function ForkMode(God) {
       cspr.process = {};
       cspr.process.pid = cspr.pid;
       cspr.pm2_env = pm2_env;
-      cspr.pm2_env.status = cst.ONLINE_STATUS;
```

Comment:
When the application is started, the status is assigned only after the application is online, not before.